ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -603,6 +603,20 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-guest-longduration-f7
```

Comment:
is it proper to set its frequency to f7 ? Usually we set it as f28

Reply:
btw, could you add 4.19 for this too?

Reply:
For 4.18 development release, we need to run f7, after that, reduce the fequence.

Reply:
Added for 4.19